dev/archery/archery/bot.py

```diff
@@ -68,13 +68,12 @@ class Command(_CommandMixin, click.Command):
 
 
 class Group(_CommandMixin, click.Group):
-
     def command(self, *args, **kwargs):
-        kwargs.setdefault('cls', Command)
+        kwargs.setdefault("cls", Command)
```

Comment:
Is it possible to avoid the gratuitous quote changes? They are polluting history without having any usefulness.

Reply:
We can add this commit to the blame ignore file, would that be sufficient?

Reply:
I didn't know that existed, do we have a blame ignore file?

Reply:
Ideally it would not do the change at all, as that will be annoying for development (leading to spurious file reloads in the editor, etc.). But having it ignored in git blames is nice already.

Reply:
We don't have a blame ignore yet but it is pretty useful especially for machine generated changes.